buildenv/jenkins/jobs/infrastructure/omrMirror

```diff
@@ -52,12 +52,13 @@ timestamps {
                                 ).trim()
                             currentBuild.description = "${LAST_COMMIT}"
 
-                            current_sha = LAST_COMMIT.split()[0]
+                            current_sha = LAST_COMMIT.take(7)
```

Comment:
I picked 7 because the lowest git version on the workers only produces 7 by default. Keep in mind, to have a false diff, 2 commits that are essentially directly next to each other in the log would have to have the same first 7 chars of the Sha. We aren't searching the whole repo for those 7chars.

Reply:
The alternative is to re-add the second shell out to git
```
current_sha = sh (
                    script: 'git rev-parse --short=10 HEAD',
                    returnStdout: true
).trim()
```